### ISSUE TITLE
Add alt text for search icon

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/securedrop.org#866]
The search icon in the main menu on top-right corner of each page should have an alt text describing the search icon functionality.

**Comment by chigby** [comment]
I think this was fixed in #885, which added alt text support to the SVG template. Note that the alt text is present on https://staging.securedrop.org/

**Comment by SaptakS** [comment]
Correct. This was fixed while doing HTML validation fixes. Thanks for pointing!
